chainer/distributions/laplace.py

```diff
@@ -0,0 +1,118 @@
+import chainer
+from chainer.backends import cuda
+from chainer import distribution
+from chainer.functions.array import broadcast
+from chainer.functions.math import basic_math
+from chainer.functions.math import clip
+from chainer.functions.math import exponential
+from chainer.functions.math import sign
+from chainer.functions.math import sqrt
+import numpy
+
+
+class Laplace(distribution.Distribution):
+
+    """Laplace Distribution.
+
+    The probability density function of the distribution is expressed as
+
+    .. math::
+        p(x;\\mu,\\sigma) = \\frac{1}{2b}
+            \\exp\\left(-\\frac{|x-\\mu|}{b}\\right)
+
+    Args:
+        loc(:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`): Parameter of distribution representing the \
+        location :math:`\\mu`.
+        scale(:class:`~chainer.Variable` or :class:`numpy.ndarray` or \
+        :class:`cupy.ndarray`): Parameter of distribution representing the \
+        scale :math:`b`.
+    """
+
+    def __init__(self, loc, scale):
+        super(Laplace, self).__init__()
+        self.loc = chainer.as_variable(loc)
+        self.scale = chainer.as_variable(scale)
+
+    def __copy__(self):
+        return self._copy_to(Laplace(self.loc, self.scale))
```

Comment:
Could you explain `__copy__` or `_copy_to` in the docstring of `chainer.Distribution`?

Reply:
Sorry. it is not needed.